### PR TITLE
Styled js code blocks in README where missing

### PR DESCRIPTION
**Pull request by amerikan**
- just added back ticks with js language identifier were it was missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,22 +195,28 @@ Puppeteer creates its own Chromium user profile which it **cleans up on every ru
    displaying. Instead of launching in headless mode, launch a full version of
    the browser using  `headless: false`:
 
+    ```js
         const browser = await puppeteer.launch({headless: false});
```
Comment (vsemozhetbyt) [inline README.md]: You can also remove the indentation now, like this:
```suggestion
     const browser = await puppeteer.launch({headless: false});
```

**Comment by aslushnikov** [comment]
@amerikan thanks for the PR! IIRC we have to keep these like this so that it imports well on developers.google.com

@ebidel - do I get it right?

**Comment by amerikan** [comment]
@aslushnikov didn't know, feel free to close is such the case.

**Comment by ebidel** [comment]
Yep, IIRC. Can't do GH style markdown for codeblocks for the import to work :(

**Pull request closed without merging**